tests/modules/keyspace_events.c

```diff
@@ -59,6 +61,11 @@ static int KeySpace_NotificationLoaded(RedisModuleCtx *ctx, int type, const char
 static int KeySpace_NotificationGeneric(RedisModuleCtx *ctx, int type, const char *event, RedisModuleString *key) {
     REDISMODULE_NOT_USED(type);
 
+    if (cached_time) {
+        RedisModule_Assert(cached_time == RedisModule_CachedTime());
+        RedisModule_Assert(cached_time != RedisModule_Time());
```

Comment:
on a very fast machine, this last assertion can fail.

Reply:
same microsecond? i doubt it but maybe you're right... any sggestions?

Reply:
add a short sleep before that line.